server/app/models/grid_service_health_check.rb

```diff
@@ -10,4 +10,11 @@ class GridServiceHealthCheck
 
   embedded_in :grid_service
 
-end
\ No newline at end of file
+  def is_valid?()
```

Comment:
Very small coding style issue here. I think `valid?` without brackets is more commonly used with ruby.